bp.h

```diff
@@ -102,6 +103,8 @@ class FactorGraph {
 		std::vector<std::tuple<int, std::shared_ptr<Proba>, std::shared_ptr<Proba>, std::shared_ptr<Proba>, std::shared_ptr<Proba>> > const & individuals = std::vector<std::tuple<int, std::shared_ptr<Proba>, std::shared_ptr<Proba>, std::shared_ptr<Proba>, std::shared_ptr<Proba>>>());
 	int find_neighbor(int i, int j) const;
 	void append_contact(int i, int j, times_t t, real_t lambdaij, real_t lambdaji = DO_NOT_OVERWRITE);
+	void check_neighbors(int i, int j);
```

Comment:
l'aggiunta di queste due funzioni sembrano gli unici cambiamenti "veri" in bp.h e bp.cpp. Puoi lasciare solo questo in questo PR?

grazie mille fabio